netbox/extras/views.py

```diff
@@ -1143,19 +1145,71 @@ def get(self, request, module, name, path=''):
         return redirect(f'{url}{path}')
 
 
-class ScriptResultView(generic.ObjectView):
+class ScriptResultView(TableMixin, generic.ObjectView):
     queryset = Job.objects.all()
 
     def get_required_permission(self):
         return 'extras.view_script'
 
+    def get_table(self, job, request, bulk_actions=True):
+        data = []
+        tests = None
+        table = None
+        index = 0
+        if job.data:
+            if 'log' in job.data:
+                if 'tests' in job.data:
+                    tests = job.data['tests']
+
+                for log in job.data['log']:
+                    index += 1
+                    result = {
+                        'index': index,
+                        'time': log.get('time', None),
+                        'status': log.get('status', None),
+                        'message': log.get('message', None),
+                    }
+                    data.append(result)
+
+                table = ScriptResultsTable(data, user=request.user)
+                table.configure(request)
+            else:
+                tests = job.data
+
+        if tests:
+            for method, test_data in tests.items():
+                if 'log' in test_data:
+                    for time, status, obj, url, message in test_data['log']:
+                        index += 1
```

Comment:
Same as above; we can call `enumerate()` and pass it the previous index value (if any) as the starting value.

Reply:
I didn't do this one as this second one potentially relies on the count of the first one, so you would need a start_counter defined at top then conditionally set the enumeration start to either the start_counter or the end of the previous list, so I thought index was just more readable as to what is going on.